ui/src/pages/bed-splash.tsx

```diff
@@ -163,7 +170,26 @@ export const BedSplash = () => {
                               {snakeToTitleCase(k)}
                             </td>
                             <td style={{ maxWidth: '120px' }} className="truncate">
-                              {value ?? 'N/A'}
+                              { k === 'global_sample_id' ?
+                              (Array.isArray(value) && value.length > 0)
+                              ? value.map((v, i) => (
+                                  v.includes('encode:')
+                                    ? <a key={i} href={'https://www.encodeproject.org/files/' + v.replace('encode:', '')}>{v}</a>
+                                    : v.includes('geo:')
+                                      ? <a key={i} href={'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=' + v.replace('geo:', '')}>{v}</a>
+                                      : v ?? 'N/A'
+                                )).reduce((prev, curr) => <>{prev}, {curr}</>)
+                              : value ?? 'N/A'
+                              :
+                                k === 'global_experiment_id' ?
+                                (Array.isArray(value) && value.length > 0) ? value.map((v, i) => (
+                                  v.includes('encode') ? <a key={i} href={'https://www.encodeproject.org'}>{v}</a> :
+                                  v.includes('geo:') ? <a key={i} href={'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=' + v.replace('geo:', '')}>{v}</a> :
+                                  v ?? 'N/A'
+                                )).reduce((prev, curr) => <>{prev}, {curr}</>) : value ?? 'N/A'
+                              :
+                                value ?? 'N/A'
+                              }
```

Comment:
😵‍💫😵‍💫😵‍💫 -- Any way to simplify this?

Reply:
I know, but I don't know how to od it easier ...

Reply:
Probably break up into sub-components, and then use `if-else` logic